src/main/resources/portal.properties.EXAMPLE

```diff
@@ -202,3 +202,13 @@ ucsc.build=hg19
 
 # default view in oncoprint (sample, patient (default))
 oncoprint.defaultview=patient
+
+# labels to be displayed in oncoprint "Mutation color" menu for custom annotation of driver and passenger mutations in the oncoprint.
+# Set these properties to enable the respective menu options in oncoprint:
+# oncoprint.custom_driver_annotation.binary.menu_label=Custom binary annotation
+# oncoprint.custom_driver_annotation.class.menu_label=Custom tiers
+
+# set this to false to disable the automatic selection of the custom driver and passenger mutations annotation in the oncoprint 
+# (instead of OncoKB and Hotspots). By default this is true. Setting this to false will select OncoKB and Hotspots as the default 
+# annotation source instead.
+# oncoprint.custom_driver_annotation.default=false
```

Comment:
When loading them with the correct properties, it works as described. I do have some more design questions ( @pieterlukasse ). (Point 6 most important).

1. For new users, it is not clear that a mouse hover above the purple / green icon shows the custom annotation.
2. The time it takes for the popup (above the green/purple icon) to show up, is very long. +2 seconds. Most people would have moved away by then. Can we make this instant?
3. The popup first popup (Mutation: 489A etc...) is large enough. Can we add the custom annotations here?
4. A popup in a popup sound like a bad idea to me.
5. The Classes/Tiers: I think we should use one name. Class makes the most sense to me atm because there is no underlying tiering). I know this can be customized, but let's make the example clear as well.
6. I think the BEST solution would be that every class/tier, passenger and driver get all their own colors. These colors would also be displayed in the legend below. This way the user can instantly see, which colors correspond to which class/passenger/driver. And a mouse hover would show the color, which corresponds to the VALUE of the column, not the column itself.

![screen shot 2017-08-14 at 11 35 16](https://user-images.githubusercontent.com/9624990/29266369-d0d0bbf4-80e4-11e7-91dc-492ec210f158.png)